testing_app/test/perf_test_e2e.dart

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file duplicate the behavior of `test_driver/perf_test.dart` but uses
+// e2e package to implement a host-independent test.
+
+import 'package:flutter/material.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:e2e/e2e.dart';
```

Comment:
Please sort the imports here. We have a convention on keeping the imports sorted in this repo.

Reply:
Done